test/e2e/examples.go

```diff
@@ -72,7 +72,7 @@ var _ = framework.KubeDescribe("[Feature:Example]", func() {
 
 		// this test wants powerful permissions.  Since the namespace names are unique, we can leave this
 		// lying around so we don't have to race any caches
-		framework.BindClusterRoleInNamespace(c.Rbac(), "edit", f.Namespace.Name,
+		framework.BindClusterRoleInNamespace(c.RbacV1beta1(), f.ClientPool, "edit", f.Namespace.Name,
```

Comment:
these two should cancel and be able to both be dropped:
```
UPSTREAM: revert: c502d10: <carry>: match kube rbac setup in e2es with openshift kinds
UPSTREAM: <carry>: match kube rbac setup in e2es with openshift kinds
```

Reply:
Dropped